src/NuGet.Config

```diff
@@ -0,0 +1,9 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <add key="local package" value="Caliburn.Micro.Autofac.Packaging\" />
```

Comment:
Cool. How do we *escape* from Win81 Sample runtime error "loop"? In order to work, it should actually point to a new 2.0.11 package, not published yet.